### DISCUSSION TITLE
GE meeting on June 12?

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#124] [General]
Any @delph-in/participants joining for the grammar engineering meeting this coming Monday June 12 18:30 London time?

Update: Meeting at https://fgv-br.zoom.us/my/arademaker

**Comment by danflick** [comment]
I'll be happy to join the meeting.

 Dan

**Comment by emilymbender** [comment]
That's a maybe from me -- depends on how far along I am with course grading.

**Comment by arademaker** [comment]
Me too, I will join.

**Comment by fcbond** [comment]
I'll miss I'm afraid

**Comment by olzama** [comment]
I'm assuming we can use https://fgv-br.zoom.us/my/arademaker again, @arademaker ?

**Reply by arademaker** [reply]
Yes
